### PR TITLE
fix(button): Change controller names

### PR DESCRIPTION
**Pull request by nathalielindqvist**
**Describe pull-request**  
Changed controller name "Variant" to "Mode variant" and options "On light" and "On dark" to "Primary" and "Secondary" on both native and web component Button.

**Solving issue**  
Fixes: [AB#2850](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2850)

**How to test**
1. Go to Storybook
2. Check in component Button, both Native and Web Component

### CODE REVIEW

**Review by theJohnnyMe — APPROVED** [review]
LGTM
Please mind the commit message when you "squash and merge" ;)

File: tegel/src/components/button/button-native.stories.ts [hunk 1]
```diff
@@ -53,14 +53,14 @@ export default {
       },
     },
     variant: {
```
Comment (mJarsater) [inline tegel/src/components/button/button-native.stories.ts:55]: Change this to modeVariant to reflect the prop. Same goes for other variant args also.

**Review by mJarsater — CHANGES_REQUESTED** [review]

**Comment by EmelieLitwin** [comment]
This PR is stale right?

File: tegel/src/components/button/button-native.stories.ts (hunk 1 above)
Comment (nathalielindqvist) [reply tegel/src/components/button/button-native.stories.ts]: Done in other PR, rebased onto this.

File: tegel/src/components/button/button-webcomponent.stories.ts [hunk 2]
```diff
@@ -55,14 +55,14 @@ export default {
       description: 'Size of the button',
     },
     variant: {
-      name: 'Variant',
+      name: 'Mode variant',
```
[the remark this answers is not in the document]
Comment (nathalielindqvist) [reply tegel/src/components/button/button-webcomponent.stories.ts]: Done in other PR, rebased onto this.

File: tegel/src/components/button/button-native.stories.ts [hunk 3]
```diff
@@ -117,7 +117,7 @@ export default {
     text: 'Button',
     btnType: 'Primary',
     size: 'Large',
-    variant: 'On dark',
+    variant: 'Primary',
```
[the remark this answers is not in the document]
Comment (nathalielindqvist) [reply tegel/src/components/button/button-native.stories.ts]: Done in other PR, rebased onto this.

**Pull request closed without merging**